packages/jellyfish-api-core/__tests__/category/loan/negativeInterest.test.ts

```diff
@@ -0,0 +1,494 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import BigNumber from 'bignumber.js'
+import { VaultActive } from '../../../src/category/loan'
+
+const testing = Testing.create(new MasterNodeRegTestContainer())
+
+let aliceAddr: string
+let bobVaultId: string
+let bobVaultAddr: string
+let bobVault: VaultActive
+let oracleId: string
+let timestamp: number
+const blocksPerDay = (60 * 60 * 24) / (10 * 60) // 144 in regtest
+// High interest rate so the change will be significant
+const interestRate = 5000
```

Comment:
hmm.. `5000` is quite abnormal.. preferably provide a more reality interest rate.. maybe `3.3`??

Reply:
Yea, i wanted to see a significant change with just one block added to the chain.

Reply:
noted that.. hmm.. the reason why i recommend the provide tricky figure like `3.3` or `5.6433` is to check the precision working
the `significant change ` can be tested by `c.generate(36)`